usaspending_api/disaster/v2/views/federal_account/loans.py

```diff
@@ -59,9 +49,17 @@ def queryset(self):
             "id": F("treasury_account__treasury_account_identifier"),
             "fa_description": F("treasury_account__federal_account__account_title"),
             "fa_id": F("treasury_account__federal_account_id"),
-            "obligation": Value(0, DecimalField(max_digits=23, decimal_places=2)),  # Throw-away field
-            "outlay": Coalesce(Sum("award__total_loan_value"), 0),  # Values are renamed at the end of the view
-            "total_budgetary_resources": Value(None, DecimalField(max_digits=23, decimal_places=2)),  # Throw-away field
+            "obligation": Coalesce(Sum("transaction_obligated_amount"), 0),
+            "outlay": Coalesce(
+                Sum(
+                    Case(
+                        When(self.final_submissions_query_filters, then=F("gross_outlay_amount_by_award_cpe")),
+                        default=Value(0),
+                    )
+                ),
+                0,
+            ),
+            "total_budgetary_resources": Coalesce(Sum("award__total_loan_value"), 0),  # "face_value_of_loan"
```

Comment:
I see why the one above is named oddly now. Should the annotation name just be fixed here to be named `face_value_of_loan`? If not, more comments are needed to explain the discrepancy

Reply:
roger